### PR TITLE
CLI: Use io.quarkus as platform group id

### PR DESCRIPTION
**Pull request by Sgitario**
- Disable flaky test in DevModeGreetingResourceIT.shouldDetectNewTests
- Disable amqp scenario because of https://github.com/quarkusio/quarkus/issues/18956
- Revert bump of HTMLunit dependency
- Bump Quarkus version to 2.0.3.Final
- CLI: Use io.quarkus as platform group id
- Remove unnecessary code from TracingExtensionExample example

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,4 +4,7 @@ updates:
   - package-ecosystem: "maven"
     directory: "/"
     schedule:
-      interval: "daily"
\ No newline at end of file
+      interval: "daily"
+    ignore:
+      # Later versions of HTML Unit does not work properly when finding elements remotely.
```
Comment (rsvoboda) [inline .github/dependabot.yml:9]: Any GH issue to be mentioned here ?

**Comment by rsvoboda** [comment]
`Use io.quarkus as platform group id` ... what is triggering this adjustment?

**Comment by Sgitario** [comment]
> `Use io.quarkus as platform group id` ... what is triggering this adjustment?

This change: https://github.com/quarkusio/quarkus/pull/18813
Before, it was using `io.quarkus` always. Now, it's using `io.quarkus.platform` that does not exist in 2.0.2.Final. 

With these changes, we ensure that we're using `io.quarkus` and the correct version we want to verify.

File: .github/dependabot.yml (hunk 1 above)
Comment (Sgitario) [reply .github/dependabot.yml:9]: Nope, this is something I experienced when running the continuous testing coverage. Nothing was working with the latest version.

**Comment by Sgitario** [comment]
For the native run, we hit the same issues of some dependencies are not found. Running native with OpenShift from local works fine, So I need to investigate it a bit further in a separated PR. 
For this PR, I think we're ok for review and merge.

**Comment by Sgitario** [comment]
rerun tests

**Comment by Sgitario** [comment]
The error in JVM was caused by socket timeout exception (network issue). In previous runs, it worked fine.

**Comment by Sgitario** [comment]
run tests

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**